holodeck/holodeck.py

```diff
@@ -13,7 +13,8 @@ class GL_VERSION(object):
     OPENGL3 = 3
 
 
-def make(world_name, gl_version=GL_VERSION.OPENGL4, resolution=None, verbose=False):
```

Comment:
Looks like a lot of this has removed the verbose code. Make sure the flag still exists and works